diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/WhiteSpaceRule.kt

```diff
@@ -96,11 +105,18 @@ import org.slf4j.LoggerFactory
 class WhiteSpaceRule(configRules: List<RulesConfig>) : DiktatRule(
     NAME_ID,
     configRules,
-    listOf(WRONG_WHITESPACE)
+    listOf(WRONG_WHITESPACE, LONG_LINE, WRONG_INDENTATION)
 ) {
+    private val configuration by lazy {
+        LineLength.LineLengthConfiguration(
+            configRules.getRuleConfig(LONG_LINE)?.configuration ?: emptyMap()
+        )
+    }
+    private lateinit var positionByOffset: (Int) -> Pair<Int, Int>
     @Suppress("ComplexMethod")
     override fun logic(node: ASTNode) {
         when (node.elementType) {
+            FILE -> positionByOffset = node.calculateLineColByOffset()
```

Comment:
as I can see, `positionByOffset` is used in `isNeedNewLineInOperatorReferences` only and calculate there.
can we refactor this field from class's field to method's variable to avoid keeping interim state in external variable

Reply:
Yes, you're right [f01b794](https://github.com/saveourtool/diktat/pull/1373/commits/f01b794c2b2b112dd192cdbc2571b3aa1aae5141)